Src/Web/Web.Shared.Net/SyntheticUserAgentTelemetryInitializer.cs

```diff
@@ -11,8 +11,10 @@
     /// </summary>
     public class SyntheticUserAgentTelemetryInitializer : WebTelemetryInitializerBase
     {
+        private const string SyntheticSourceNameKey = "Microsoft.ApplicationInsights.RequestTelemetry.SyntheticSource";
```

Comment:
Is there any need to have this be a human-readable string? If not, consider a static object:
```C#
private static readonly object SyntheticSourceNameKey = new object();
```
It's GetHashCode implementation will be faster and, therefore, look-ups and inserts will be too.